### ISSUE TITLE
Application error on page layer 2 when setting Portuguese language

### ISSUE THREAD

**Issue by rafarocha** [ethereum/ethereum-org-website#13245]
### Describe the bug

When opening the page about layer 2 in Portuguese Brazilian language, I get a client-side error. When trying to open the same page in another language, English or Italian, it all works fine.

### To reproduce

1. Go to page [`ethereum.org`](ethereum.org) with English Language
2. Click on the search icon at the top
3. Type `layer 2`
4. All of it works fine but ...
5. Select on the top `Portuguese Brazilian` language
6. See error

### Expected behavior

Expected to see the info page about Ethereum on layer 2 with success as an English language https://ethereum.org/en/layer-2/

### Screenshots

<img width="1137" alt="Screenshot 2024-06-25 at 22 08 47" src="https://github.com/ethereum/ethereum-org-website/assets/730268/b35245dc-b34e-493d-8f3b-eb174cdabdaf">

### Desktop (please complete the following information)

- MacOS Sonoma 14.0
- Chrome 126.0.6478.62 (Official Build) (arm64)

### log1
```log 
DOMException: Failed to execute 'setAttribute' on 'Element': ']' is not a valid attribute name.
    at o (https://ethereum.org/_next/static/chunks/pages/_app-b1445fbd6e2a82e9.js:1:21311)
    at https://ethereum.org/_next/static/chunks/pages/_app-b1445fbd6e2a82e9.js:1:22499
    at https://ethereum.org/_next/static/chunks/pages/_app-b1445fbd6e2a82e9.js:1:22507
    at Array.reduce (<anonymous>)
    at e (https://ethereum.org/_next/static/chunks/pages/_app-b1445fbd6e2a82e9.js:1:22075)
    at https://ethereum.org/_next/static/chunks/pages/_app-b1445fbd6e2a82e9.js:1:23328
    at Array.map (<anonymous>)
    at https://ethereum.org/_next/static/chunks/pages/_app-b1445fbd6e2a82e9.js:1:21691
    at t.Z (https://ethereum.org/_next/static/chunks/pages/_app-b1445fbd6e2a82e9.js:20:53975)
    at ak (https://ethereum.org/_next/static/chunks/framework-56eb74ff06128874.js:9:60903)
a9 @ framework-56eb74ff06128874.js:9
```

### log2
```log 
DOMException: Failed to execute 'setAttribute' on 'Element': ']' is not a valid attribute name.
at o (https://ethereum.org/_next/static/chunks/pages/_app-b1445fbd6e2a82e9.js:1:21311)
    at https://ethereum.org/_next/static/chunks/pages/_app-b1445fbd6e2a82e9.js:1:22499
    at https://ethereum.org/_next/static/chunks/pages/_app-b1445fbd6e2a82e9.js:1:22507
    at Array.reduce (<anonymous>)
    at e (https://ethereum.org/_next/static/chunks/pages/_app-b1445fbd6e2a82e9.js:1:22075)
    at https://ethereum.org/_next/static/chunks/pages/_app-b1445fbd6e2a82e9.js:1:23328
    at Array.map (<anonymous>)
    at https://ethereum.org/_next/static/chunks/pages/_app-b1445fbd6e2a82e9.js:1:21691
    at t.Z (https://ethereum.org/_next/static/chunks/pages/_app-b1445fbd6e2a82e9.js:20:53975)
    at ak (https://ethereum.org/_next/static/chunks/framework-56eb74ff06128874.js:9:60903)
```

### log3
```log 
A client-side exception has occurred, see here for more info: https://nextjs.org/docs/messages/client-side-exception-occurred
```

### log4
```log
Uncaught Error: Minified React error #423; visit https://reactjs.org/docs/error-decoder.html?invariant=423 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at i (framework-56eb74ff06128874.js:9:120775)
    at oD (framework-56eb74ff06128874.js:9:99073)
    at framework-56eb74ff06128874.js:9:98940
    at oO (framework-56eb74ff06128874.js:9:98947)
    at oE (framework-56eb74ff06128874.js:9:95699)
    at ox (framework-56eb74ff06128874.js:9:94254)
    at x (framework-56eb74ff06128874.js:33:1364)
    at MessagePort.T (framework-56eb74ff06128874.js:33:1894)
```

### log5
```log
warning: _app-b1445fbd6e2a82e9.js:1 react-i18next:: 
You will need to pass in an i18next instance by using initReactI18next
```

### log6 
```log
Manifest: Line: 42, column: 1, Unexpected token.
```

### log7
```log
The resource https://ethereum.org/fonts/inter/latin.woff2 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.Understand this warning
layer-2/:1 
```

### Would you like to work on this issue?

- [ ] Yes
- [ ] No

**Comment by wackerow** [comment]
Hey @rafarocha, thanks for reporting... I'm unable to reproduce with the steps you've provided, would you mind adding a little clarity?

With your instructions, I'm going to https://ethereum.org, clicking the search and typing "layer 2" but then your instructions becomes less clear. Are you navigating to the first "layer 2" search result? Are you closing the modal?

> Select on the top Portuguese Brazilian language

Not exactly sure what is being referred to here... is this inside the language picker menu?

Links https://ethereum.org/en/layer-2/ and https://ethereum.org/pt-br/layer-2/ are both working for me. Let us know, thanks!

**Comment by rafarocha** [comment]
Yes. But I understood now. 
I got confused with the language Portuguese instead of Brazilian Portuguese. 

Follow the steps:
1. Go to https://ethereum.org/en/layer-2
2. Click on the languages ​​menu to show the list of languages
3. Type "Portuguese"
4. Select the "Portuguese" language as shown in the image.

This problem also occurs in any browser.
For the Brazilian Portuguese language, this bug does not occur.

<img width="1099" alt="Screenshot 2024-06-30 at 11 04 50" src="https://github.com/ethereum/ethereum-org-website/assets/730268/48d846e3-7d37-4d07-8c89-14425283d1cd">

**Comment by wackerow** [comment]
Got it! Thanks for clarifying @rafarocha, I am indeed now able to reproduce this, getting the infamous `Application error: a client-side exception has occurred (see the browser console for more information).` error that you described above:

<img width="967" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/6febec6b-207d-41e2-ab42-6b69cf25d99c">

**I'm actually getting this by simply going to https://ethereum.org/pt/layer-2/**

cc: @pettinarip Want to make you aware of this. Pointing to an attribute being set incorrectly in the errors, but not clear which it could be... Only seeing a couple places where we use `setAttribute`:

<img width="522" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/a1e25671-e87f-4e1b-8b1e-580c957ce1f3">

Any thoughts what could be going on here?

**Comment by pettinarip** [comment]
@wackerow took a look, the error is coming from the translations. There is an extra `]` that makes `htmr` to fail parsing it.

![image](https://github.com/ethereum/ethereum-org-website/assets/468158/3ecc732d-e48d-4797-b82b-48ad530b6cb3)

Pushing a hotfix now.

**Comment by pettinarip** [comment]
Pushed a hot fix #13296

**Comment by wackerow** [comment]
Closed with #13296
Thanks everyone!
